Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -13,14 +14,16 @@ class TestIncrementalPageRank(TestCase):
     As it is a Monte Carlo method, integration tests are often probabilistic in nature.
     That is, they might fail on rare occasions.
     """
-    def create_graph(self, nr_of_nodes, nr_of_edges, nr_of_times=1):
+    def create_graph(self, nr_of_nodes, nr_of_edges, nr_of_times=1, directed=True):
```

Comment:
I think we can require the graph to be directed, so you can remove this optional argument.